libraries/GCS_MAVLink/GCS_config.h

```diff
@@ -135,3 +135,7 @@
 #ifndef AP_MAVLINK_MAV_CMD_SET_HAGL_ENABLED
 #define AP_MAVLINK_MAV_CMD_SET_HAGL_ENABLED (BOARD_FLASH_SIZE > 1024)
 #endif
+
+#ifndef AP_MAVLINK_MSG_FLIGHT_INFORMATION_ENABLED
+#define AP_MAVLINK_MSG_FLIGHT_INFORMATION_ENABLED 1
```

Comment:
What's the flash size cost of this new feature?

Will it cause us to start overflowing on smaller boards?

Should we add it into `minimize_common.inc`? (@andyp1per and @Hwurzburg might have ideas on that)

We don't need to change this as part of this PR.

Reply:
> What's the flash size cost of this new feature?

~728 bytes.

```
          Target          Text (B)  Data (B)  BSS (B)  Total Flash Used (B)  Free Flash (B)  External Flash Used (B)
          ----------------------------------------------------------------------------------------------------------
Disabled: bin/arducopter   1840268      3920   258372               1844188          121876  Not Applicable 
Enabled:  bin/arducopter   1840996      3920   258372               1844916          121148  Not Applicable  
```